diff -urN gnome-logs-3.11.2.old/aclocal.m4 gnome-logs-3.11.2/aclocal.m4
--- gnome-logs-3.11.2.old/aclocal.m4	2013-11-18 22:40:00.000000000 +0000
+++ gnome-logs-3.11.2/aclocal.m4	2013-12-12 19:16:33.396241488 +0000
@@ -1385,7 +1385,7 @@
 mostlyclean-am: clean-appdata-xml
 
 %.appdata.valid: %.appdata.xml
-	$(AM_V_GEN) if test -f "$<"; then d=; else d="$(srcdir)/"; fi; $(APPDATA_VALIDATE) $${d}$< && touch [$]@
+	$(AM_V_GEN) if test -f "$<"; then d=; else d="$(srcdir)/"; fi; $(APPDATA_VALIDATE) --nonet $${d}$< && touch [$]@
 
 all-am: $(appdata_XML:.appdata.xml=.appdata.valid)
 uninstall-am: uninstall-appdata-xml
diff -urN gnome-logs-3.11.2.old/configure gnome-logs-3.11.2/configure
--- gnome-logs-3.11.2.old/configure	2013-11-18 22:40:00.000000000 +0000
+++ gnome-logs-3.11.2/configure	2013-12-12 19:21:14.491248666 +0000
@@ -4618,7 +4618,7 @@
 mostlyclean-am: clean-appdata-xml
 
 %.appdata.valid: %.appdata.xml
-	$(AM_V_GEN) if test -f "$<"; then d=; else d="$(srcdir)/"; fi; $(APPDATA_VALIDATE) $${d}$< && touch $@
+	$(AM_V_GEN) if test -f "$<"; then d=; else d="$(srcdir)/"; fi; $(APPDATA_VALIDATE) --nonet $${d}$< && touch $@
 
 all-am: $(appdata_XML:.appdata.xml=.appdata.valid)
 uninstall-am: uninstall-appdata-xml
